### PR TITLE
Updating irys to latest

### PR DESCRIPTION
**Pull request by blockiosaurus**

### CODE REVIEW

File: packages/umi-uploader-irys/package.json [hunk 1]
```diff
@@ -28,9 +28,9 @@
     "test": "ava"
   },
   "dependencies": {
-    "@irys/sdk": "^0.0.2",
+    "@irys/sdk": "0.2.10",
```
Comment (lorisleiva) [inline packages/umi-uploader-irys/package.json:31]: Any reason for removing the `^` prefix which makes the version range follow semver?

**Review by lorisleiva — APPROVED** [review]
Thanks!

File: packages/umi-uploader-irys/package.json (hunk 1 above)
Comment (blockiosaurus) [reply packages/umi-uploader-irys/package.json:31]: Nope, my extension just removes them apparently.

Comment (lorisleiva) [reply packages/umi-uploader-irys/package.json]: Gotcha. Same issue for `bignumber.js` btw.

Comment (blockiosaurus) [reply packages/umi-uploader-irys/package.json]: Thanks! Gonna have to see if I can turn that setting off 😅

**Pull request merged**